.circleci-disabled/config.yml

```diff
@@ -0,0 +1,373 @@
+version: 2.1
```

Comment:
Something went wrong when you rebased/squashed your commits. This file was deleted in #497 and your change is putting it back.